lisa/schema.py

```diff
@@ -1450,6 +1450,8 @@ class Platform(TypedSchema, ExtendableSchemaMixin):
     # capture kernel config info or not
     capture_kernel_config_information: bool = False
     capture_vm_information: bool = True
+    # capture features settings
```

Comment:
This comment is useless. You can explain where the feature settings are captured to.